src/lang/english.txt

```diff
@@ -194,6 +194,7 @@ STR_COLOUR_DEFAULT                                              :Default
 STR_UNITS_VELOCITY_IMPERIAL                                     :{COMMA}{NBSP}mph
 STR_UNITS_VELOCITY_METRIC                                       :{COMMA}{NBSP}km/h
 STR_UNITS_VELOCITY_SI                                           :{COMMA}{NBSP}m/s
+STR_UNITS_VELOCITY_PLAYORIENTED                                 :{DECIMAL}{NBSP}tiles/day
```

Comment:
I'm not a huge fan of "play oriented" as a string, but I can't think of any good alternatives. "Friendly", perhaps?

Reply:
"Player's units"?

Reply:
"In-game", maybe.

Reply:
"Simulation"?

Reply:
"Abstract"?

Reply:
"Tiles per day"?  Name the thing for the thing?

Reply:
my choice would be "game units"

Reply:
@andythenorth That would be inconsistent with the other strings in the same drop-down (e.g., "Imperial" rather than "miles per hour").

I've gone ahead and made the change to "Game units".